backend_site/projects/views.py

```diff
@@ -0,0 +1,14 @@
+"""API routes for the Projects record type"""
+
+from flask import Blueprint
+from flask_cors import CORS
+
+blueprint = Blueprint('projects', __name__)
+CORS(blueprint)
+
+
+@blueprint.route('', methods=['GET'])
+def getProjects():
+    """Route to retrieve all LUG projects, past and present"""
+    with open('projects.json') as projects_file:
+        return projects_file.read()
```

Comment:
I'm getting a weird issue where pinging the `/api/projects` endpoint gives me the JSON data in `text/html` instead of `application/json` like the rest of the endpoints.

Do you have any possible fixes for this? The other endpoints use `jsonify` while this one returns the file data directly.

Reply:
I wrapped in a jsonify I'll test and get this pushed up shortly

Reply:
Fixed and tested if you want to pull latest